packages/test-utils/src/wrapper.js

```diff
@@ -420,8 +420,16 @@ export default class Wrapper implements BaseWrapper {
     }
 
     Object.keys(data).forEach((key) => {
-      // $FlowIgnore : Problem with possibly null this.vm
-      this.vm.$set(this.vm, [key], data[key])
+      if (typeof data[key] === 'object' && data[key] !== null) {
```

Comment:
If `data[key]` is an object, `data[key]` is not null.
I think `if (typeof data[key] === 'object') {` is better.